docs/backends/xnvme_info_zoned.out

```diff
@@ -0,0 +1,43 @@
+
+
+
+
+
+
```

Comment:
Same here

Reply:
It is the raw output from the command in the ``.cmd`` file. If the commands output empty lines, then there will be empty lines here.